client/lib/login/index.js

```diff
@@ -158,6 +159,10 @@ export function getSignupUrl( currentQuery, currentRoute, oauth2Client, locale,
 		return `${ signupUrl }/account?${ params.toString() }`;
 	}
 
+	if ( locale && locale !== 'en' ) {
```

Comment:
Nit: Can be replaced by [isDefaultLocale()](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/utils.ts#L32).

Reply:
Making this change breaks [this test](https://github.com/Automattic/wp-calypso/blob/b995b1478e6a05fe0a6217d37eeb7d7fc0870da8/client/lib/login/test/index.js#L39).

Reply:
Is the change not working as expected only in the test environment, or is it also not working in development or/and production?

If it only fails in tests, but works as intended in production, it would make sense to update the unit test, rather than blocking the changes.

Reply:
I agree with @yuliyan - would be nice if we don't hardcode the default locale when we have a utility function available for that.